src/AEG-IO.ts

```diff
@@ -48,35 +54,62 @@ export async function saveFile(handle: FileSystemFileHandle, aegData: AEGTree) {
 
 /**
  * Function that takes in data read from a file and converts it into a valid AEG representation.
+ * @param mode The mode we are in (Draw mode or proof mode)
  * @param fileData The data read from a file.
- * @returns An AEG representation of the data.
+ * @returns If in draw mode, returns an AEG representation of the data.
+ * If in proof mode, constructs an array of AEGs read from the file.
+ * This can be used to build the proof list
  * Returns null if an error occurred
  */
-export function loadFile(fileData: string | ArrayBuffer | null): AEGTree | null {
+export function loadFile(
+    mode: "Draw" | "Proof",
+    fileData: string | ArrayBuffer | null
```

Comment:
Why are we supporting ArrayBuffers as an input type for fileData if we're just going to return null if an array buffer is passed in?

Reply:
Oh I think I just left this in earlier when I first did save and load and forgot to check for within index, my bad